lib/addons/neuron/index.ts

```diff
@@ -0,0 +1,31 @@
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { loadYaml, readYamlDocument } from "../../utils/yaml-utils";
+
+export class NeuronPluginAddOn implements ClusterAddOn {
+    deploy(clusterInfo: ClusterInfo): void {
```

Comment:
Return type is `void` - you don't want to return something here, so that other add-ons can depend on it?

Reply:
returning promise